pipelines/broad/dna_seq/germline/joint_genotyping/reblocking/test_inputs/exome/Scientific/D5327.NA12891.json

```diff
@@ -1,4 +1,7 @@
 {
   "ReblockGVCF.gvcf": "gs://broad-gotc-test-storage/germline_single_sample/exome/scientific/truth/{TRUTH_BRANCH}/D5327.NA12891/NA12891.g.vcf.gz",
   "ReblockGVCF.gvcf_index": "gs://broad-gotc-test-storage/germline_single_sample/exome/scientific/truth/{TRUTH_BRANCH}/D5327.NA12891/NA12891.g.vcf.gz.tbi"
+  "ReblockGVCF.ref_fasta": "gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.fasta",
```

Comment:
Not sure if this is just a weird github thing, but I am seeing the ":" on this line highlighted in red. Not sure what that means, if anything

Reply:
Looks like a missing comma after D5327.NA12891/NA12891.g.vcf.gz.tbi"